Apps/AdminWebClient/src/ClientApp/src/services/interfaces.ts

```diff
@@ -48,6 +48,7 @@ export interface IDashboardService {
 export interface ICommunicationService {
   initialize(http: IHttpDelegate): void;
   add(communication: Communication): Promise<void>;
+  getBannerList(): Promise<Communication[]>;
```

Comment:
will this one return only banner communications or all coms?

Reply:
great question! I will need to check with @sam-warren to make sure what the real service will return here.

Reply:
The backend service just returns all items in the Communication table.